pkg/resources/factory.go

```diff
@@ -119,3 +131,13 @@ func (rf *resourceFactory) GetResourcePool(rc *types.ResourceConfig, deviceList
 	rPool := newResourcePool(rc, apiDevices, devicePool)
 	return rPool, nil
 }
+
+func (rf *resourceFactory) GetRdmaSpec(vendor, pciAddrs string) types.RdmaSpec {
+	switch vendor {
+	// Mellanox device
+	case "15b3":
```

Comment:
this case should not be there either. it should be just based on the pciAddr

Reply:
@paravmellanox how is the device plugin going to get the rdmaDeviceSpec from the PCI address?